### PR TITLE
Return multiple licenses.

### PR DESCRIPTION
**Pull request by isoos**
(Detection change will be part of a subsequent PR.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 * Greatly expanded and improved license detection.
 
+## [next release]
```
Comment (kevmoo) [inline CHANGELOG.md]: update pubspec to be 0.5.0-dev – then make this `0.5.0`

File: test/license_test.dart [hunk 2]
```diff
@@ -35,7 +35,7 @@ main() {
 
   group('BSD', () {
```
Comment (kevmoo) [inline test/license_test.dart]: Do we have a test for package w/ multiple licenses?

Comment (isoos) [reply test/license_test.dart]: I thought I have, but I can't recall it. I remember a license file that has two licenses copied into it.

Comment (kevmoo) [reply test/license_test.dart]: I don't see a test result that exercises it. Would be nice...

Comment (isoos) [reply test/license_test.dart]: I'll add this with the next change that will also include the file name in the license object.

**Pull request merged**